src/ui/ViewModels/OrganisationViewModel.cs

```diff
@@ -58,8 +58,18 @@ public OrganisationViewModel()
         [Required(ErrorMessage = "Enter your county")]
         public string Addr4 { get; set; }
 
-        [Required(ErrorMessage = "Enter postcode")]
-        public string Postcode { get; set; }
+        [MinLength(7, ErrorMessage = "Postcode is too short. Enter a postcode in the format ‘SW10 1AA’")]
```

Comment:
Do we know what the minimum is? The design says 7 is the maximum if we don't include the space.

Reply:
This is wrong actually. The minimum should be 5

Reply:
5 probably has to be the minimum, btw. e.g. `E16DS`